Assets/Scripts/GameController.cs

```diff
@@ -97,12 +96,12 @@ public class GameController : ParentController {
 		myUnits = new Dictionary<int, Unit>();
 		enemyUnits = new Dictionary<int, Unit>();
 		PlacingUnits = !UnitsArePlaced;
-		_currentMap = GameData.GetMap(GameData.CurrentMatch.MapName);
+		_currentMap = GameData.GetMaps[GameData.CurrentMatch.MapName];
```

Comment:
GetMaps should be a function and not look like a direct property access in my opinion.